### ISSUE TITLE
Training runs but skips all epochs

### ISSUE THREAD

**Issue by LordOfLuck** [facebookresearch/wav2letter#752]
I am trying to finetune a pretrained model on custom data. The training starts, but i am getting very very fast Epoch started and shuffling dataset notifications and no logs in the logfile.
```
I0716 07:31:26.996534  5724 Train.cpp:571] Epoch 520401 started!
I0716 07:31:26.996552  5724 Train.cpp:564] Shuffling trainset
I0716 07:31:26.996563  5724 Train.cpp:571] Epoch 520402 started!
I0716 07:31:26.996587  5724 Train.cpp:564] Shuffling trainset
I0716 07:31:26.996603  5724 Train.cpp:571] Epoch 520403 started!
I0716 07:31:26.996628  5724 Train.cpp:564] Shuffling trainset
I0716 07:31:26.996641  5724 Train.cpp:571] Epoch 520404 started!
I0716 07:31:26.996665  5724 Train.cpp:564] Shuffling trainset
I0716 07:31:26.996681  5724 Train.cpp:571] Epoch 520405 started!
I0716 07:31:26.996711  5724 Train.cpp:564] Shuffling trainset
I0716 07:31:26.996723  5724 Train.cpp:571] Epoch 520406 started!
I0716 07:31:26.996743  5724 Train.cpp:564] Shuffling trainset
I0716 07:31:26.996763  5724 Train.cpp:571] Epoch 520407 started!
I0716 07:31:26.996790  5724 Train.cpp:564] Shuffling trainset
I0716 07:31:26.996803  5724 Train.cpp:571] Epoch 520408 started!
I0716 07:31:26.996827  5724 Train.cpp:564] Shuffling trainset
I0716 07:31:26.996840  5724 Train.cpp:571] Epoch 520409 started!
I0716 07:31:26.996856  5724 Train.cpp:564] Shuffling trainset
I0716 07:31:26.996871  5724 Train.cpp:571] Epoch 520410 started!
I0716 07:31:26.996888  5724 Train.cpp:564] Shuffling trainset
```
My training command looks as follows:
```
wav2letter/build/Train fork online-asr/training/am/am_tds_ctc_librivox_dev_other.bin --flagsfile online-asr/training/am/train_tds_ctc.cfg --minloglevel=0 --logtostderr=1
```

What could be the error? It seems like it completely ignores the data.
At the begginning I am getting `Falling back to using letters as targets for the unknown word:` for many words in my dataset, but then letters should be used as targets.

**Comment by tlikhomanenko** [comment]
Could you post full log you have during the run and also you config file?

I suspect all the data just filtered, you can check this in the log if this happens.

**Comment by vineelpratap** [comment]
(closing due to inactivity)
